examples/virt-cirros.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: virtlet-example-cirros
+spec:
+  containers:
+    - name: cirros
+      image: 172.19.0.1/cirros
```

Comment:
IMO Docs should have an info where this ip comes from and how this should be set.